iree/compiler/Codegen/SPIRV/KernelDispatchUtils.cpp

```diff
@@ -498,9 +498,27 @@ static LogicalResult setMaliSpecificConfig(
     SmallVector<int64_t, 4> fourthLevel = {0, 0, 0, 0, 1, 1, 4};
     tileSizes.emplace_back(fourthLevel);
 
-    return setOpConfigAndEntryPointFnTranslation(
-        entryPoint, op, tileSizes, /*nativeVectorSize=*/ArrayRef<int64_t>{},
-        IREE::HAL::DispatchLoweringPassPipeline::SPIRVVectorize, workgroupSize);
+    if (failed(setOpConfigAndEntryPointFnTranslation(
+            entryFn, op, tileSizes, /*nativeVectorSize=*/ArrayRef<int64_t>{},
+            IREE::HAL::DispatchLoweringPassPipeline::SPIRVVectorize,
+            workgroupSize)))
+      return failure();
+
+    // Let the entry point region to return fully static number of workgroups.
```

Comment:
Please add a TODO here that needs to be dropped when affine min generation in Tiling is fixed.